urbanairship-react-native/ios/UARCTModule/UARCTEventEmitter.m

```diff
@@ -260,13 +267,11 @@ - (NSMutableDictionary *)eventBodyForNotificationResponse:(UANotificationRespons
     return body;
 }
 
-+ (NSMutableDictionary *)eventBodyForNotificationContent:(UANotificationContent *)content {
++ (NSMutableDictionary *)eventBodyForNotificationContent:(NSDictionary *)userInfo notificationIdentifier:(NSString *)identifier {
     NSMutableDictionary *pushBody = [NSMutableDictionary dictionary];
-    [pushBody setValue:content.alertBody forKey:@"alert"];
-    [pushBody setValue:content.alertTitle forKey:@"title"];
```

Comment:
I have some doubt about this one. Should we still need the expose all the content or just extras ? @ralepinski

Reply:
In fact, since we updated the `UAPushDelegate` we don't exhibit the notification content anymore, we only exhibit the user information so we lost some informations like the `alertBody`,  `alertTitle` and the `identifier`.

The new delegate
```
-(void)receivedForegroundNotification:(NSDictionary *)userInfo completionHandler:(void (^)(void))completionHandler {

}
```